src/AutoEnrollPlugin/Service/StreamMappingService.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+using Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Models;
+using Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Factories;
+using System.Collections.Generic;
+
+namespace Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Services
+{
+    public class StreamMappingService : IStreamMappingService
```

Comment:
Everything is called service. It is hard to reason about what each class does when everyhing is service. For example this to me seems more like `Stream Rule/Configuration Resolver/Factory` or similar